src/project/internal/projectactionscontroller.cpp

```diff
@@ -622,6 +626,67 @@ void ProjectActionsController::clearRecentProjects()
     recentFilesController()->clearRecentFiles();
 }
 
+bool ProjectActionsController::shouldRetryLoadAfterError(const Ret& ret, const muse::io::path_t& filepath)
+{
+    if (ret) {
+        return true;
+    }
+    //! TODO AU4 :
+    switch (static_cast<project::Err>(ret.code())) {
+    /*
```

Comment:
Do we need this commented block here? It is copied from musescore. FileTooOld, TooNew or Old300Format have no relevance to audacity. The same goes for projecterrors.h